.codecov.yml

```diff
@@ -7,6 +7,17 @@ codecov:
     - !travis
 
 coverage:
+  precision: 3
+  round: down
+  range: 80...100
+
+  status:
+    # Learn more at https://codecov.io/docs#yaml_default_commit_status
+    project: true
+    patch: true
+    changes: false
```

Comment:
I have tested it over here https://github.com/wilzbach/phobos/pull/8

![image](https://cloud.githubusercontent.com/assets/4370550/17462175/bf88f998-5ca4-11e6-9d52-1f31cfcb1272.png)

patch: the coverage of the current diff (should error if it's lower than the overall coverage)
project: the overall coverage (should error if it decreases)
changes:  unexpected changes outside of the diff, afaict project - patch